### PR TITLE
clean quoref and add prompts

### PR DESCRIPTION
**Pull request by zaidalyafeai**
Added more prompts to reach 10 original task templates. There are two prompts that are non-original task and might be better to remove 

- Predict URL
- Predict Answer Index

### CODE REVIEW

File: promptsource/templates/quoref/templates.yaml [hunk 1]
```diff
@@ -1,5 +1,25 @@
 dataset: quoref
 templates:
+  4120bebc-9c8f-44af-8d1a-a65e443ce010: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 4120bebc-9c8f-44af-8d1a-a65e443ce010
+    jinja: 'The answer to the question : {{question}} is inside the article: {{context}},
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:7]:
```suggestion
    jinja: 'The answer to the question: {{question}} is inside the article: {{context}},
```

File: promptsource/templates/quoref/templates.yaml [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+      can you guess it ?
+
+
+      |||
+
+      {{answers.text[0]}}
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:13]:
```suggestion
      {{answers.text | choice}}
```

File: promptsource/templates/quoref/templates.yaml [hunk 3]
```diff
@@ -19,10 +39,41 @@ templates:
...
+      metrics:
+      - Squad
+      original_task: true
+    name: Find Answer
+    reference: ''
+  a3e5e25d-0a87-4cb8-89ab-3539fc4d23cb: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: a3e5e25d-0a87-4cb8-89ab-3539fc4d23cb
+    jinja: 'This article: {{context}} contains an answer for the question : {{question}},
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:65]:
```suggestion
    jinja: 'This article: {{context}} contains an answer for the question: {{question}},
```

File: promptsource/templates/quoref/templates.yaml [hunk 4]
```diff
@@ -19,10 +39,41 @@ templates:
...
+      what is it ?
+
+      |||
+
+      {{answers.text[0]}}'
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:70]:
```suggestion
      {{answers.text | choice}}'
```

File: promptsource/templates/quoref/templates.yaml [hunk 5]
```diff
@@ -40,10 +91,31 @@ templates:
...
+    answer_choices: null
+    answer_choices_key: null
+    id: abdfa570-2de5-406c-9051-caa6a1362796
+    jinja: 'What is the answer for the question: {{question}} from the following article
+      ?
+
+
+      {{context}}|||
+
+      {{answers.text[0]}}
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:110]:
```suggestion
      {{answers.text | choice}}
```

File: promptsource/templates/quoref/templates.yaml [hunk 6]
```diff
@@ -65,10 +137,29 @@ templates:
...
+      - Other
+      original_task: false
+    name: Predict Answer Index
+    reference: ''
+  b3ec0888-dd6f-466a-abd4-b2fbcacfdb8b: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: b3ec0888-dd6f-466a-abd4-b2fbcacfdb8b
+    jinja: 'I have a test where I am given the following article, what is an answer
+      for the question : {{question}} ?
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:151]:
```suggestion
      for the question: {{question}} ?
```

File: promptsource/templates/quoref/templates.yaml [hunk 7]
```diff
@@ -65,10 +137,29 @@ templates:
...
+
+
+      {{context}}|||
+
+      {{answers.text[0]}}'
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:156]:
```suggestion
      {{answers.text | choice}}'
```

File: promptsource/templates/quoref/templates.yaml [hunk 8]
```diff
@@ -86,10 +177,51 @@ templates:
...
+  d055747f-7a32-4e12-aab1-fed35d42a445: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: d055747f-7a32-4e12-aab1-fed35d42a445
+    jinja: 'Found the following article online, use it to answer the question : {{question}}
+
+
+      {{context}}|||
+
+      {{answers.text[0]}}
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:196]:
```suggestion
      {{answers.text | choice}}
```

File: promptsource/templates/quoref/templates.yaml [hunk 9]
```diff
@@ -86,10 +177,51 @@ templates:
...
+      - ROUGE
+      - BLEU
+      original_task: false
+    name: Guess Title For Context
+    reference: ''
+  d055747f-7a32-4e12-aab1-fed35d42a445: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: d055747f-7a32-4e12-aab1-fed35d42a445
+    jinja: 'Found the following article online, use it to answer the question : {{question}}
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:191]:
```suggestion
    jinja: 'Found the following article online, use it to answer the question: {{question}}
```

File: promptsource/templates/quoref/templates.yaml [hunk 10]
```diff
@@ -86,10 +177,51 @@ templates:
...
+    answer_choices: null
+    answer_choices_key: null
+    id: d1abb8a0-03c4-41ef-865c-aa275278a0e4
+    jinja: 'A friend asked me to answer this question: {{question}}, using the article:
+      {{context}}, what would be the answer ?
+
+
+      |||
+
+      {{answers.text[0]}}
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:216]:
```suggestion
      {{answers.text | choice}}
```

File: promptsource/templates/quoref/templates.yaml [hunk 11]
```diff
@@ -104,8 +236,28 @@ templates:
...
+    answer_choices: null
+    answer_choices_key: null
+    id: fcbe0609-06ce-4cbd-91de-adc38966bcac
+    jinja: 'Read the following paragraph and extract the answer for the question:
+      {{question}}
+
+
+      {{context}} |||
+
+      {{answers.text[0]}}'
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:256]:
```suggestion
      {{answers.text | choice}}'
```

File: promptsource/templates/quoref/templates.yaml [hunk 12]
```diff
@@ -19,10 +39,41 @@ templates:
...
+      - Squad
+      original_task: true
+    name: Answer Question Given Context
+    reference: ''
+  9493f80a-daf5-4c30-a9fc-7bc5bc61b5e9: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 9493f80a-daf5-4c30-a9fc-7bc5bc61b5e9
+    jinja: "The following article contains an answer for the question: {{question}}\
+      \ , can you please find it? \n\n{{context}}|||\n{{answers.text[0]}}"
```
Comment (VictorSanh) [inline promptsource/templates/quoref/templates.yaml:53]:
```suggestion
      \ , can you please find it? \n\n{{context}}|||\n{{answers.text | choice}}"
```

**Comment by VictorSanh** [comment]
corrected some typos and jinja thing. merging now

`Predict URL` and `Predict Answer Index` could you flag them on the spreadsheet. also could you give more details in the spreadsheet about the metric for `Predict Answer Index` as you put `Other`? @zaidalyafeai

**Comment by zaidalyafeai** [comment]
@VictorSanh I didn't add this the Predict Answer Index prompt and I have no idea what metric to use for it. I find it weird to predict the index of the answer instead of the answer itself.

**Comment by VictorSanh** [comment]
> @VictorSanh I didn't add this the Predict Answer Index prompt and I have no idea what metric to use for it. I find it weird to predict the index of the answer instead of the answer itself.

no worries, we decided to remove them

**Pull request merged**